content/influxdb/cloud/account-management/switch-org.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Switch InfluxDB Cloud organizations
+seotitle: Switch between InfluxDB Cloud organizations
+description: >
+  Switch from one InfluxDB Cloud account to another and set a default account.  
+menu:
+  influxdb_cloud:
+    name: Switch InfluxDB accounts
+    parent: Account management
+weight: 105
+---
+If you belong to more than one {{< cloud-name >}} organization with the same email address, you can switch from one organization to another while staying logged in. An account can contain multiple organizations. 
+
+To switch {{< cloud-name "short" >}} organizations:
+
+1. In the {{< cloud-name "short" >}} UI, click the organization name dropdown in the upper left and select **Settings**.
+2. Click **Switch Organization**. If this option doesn't appear, your email address is only associated with one organization.
```

Comment:
@gunnaraasen -- Can you confirm these steps? wasn't able to try it out with multiple orgs.

Reply:
This is incorrect. Steps are: 

To switch {{< cloud-name "short" >}} organizations:

1. In the {{< cloud-name "short" >}} UI, click the organization name dropdown in the upper left and select **Switch Organizations**.
2. Then select the organization that you would like to switch to. 

Note: If the organization is in a different account, you must switch accounts first in order to view the account's corresponding organizations. 

@noramullen1